### PR TITLE
Page/main

### PR DESCRIPTION
**Pull request by daleinen7**
This is for the homepage. 

In theory I just need to import existing components and fill in the needed content. In practice there was much work needed. 

I did a few things in addition to importing the components and filling in the props to make the page:
1. Line heights of fonts needed adjusted
2. Styled tile icons (currently only works for 4 but I know there are a few 3 column tile grids needed on other pages)
3. Refactor content section to allow for images to alternate order

### CODE REVIEW

File: src/components/Nav.jsx
Comment (diorshelton) [inline src/components/Nav.jsx:1]: Should we use the Button component in this file for the Donate button?

File: src/pages/index.jsx [hunk 1]
```diff
@@ -1,21 +1,85 @@
...
+        }
+        linkText={'Join us on Slack'}
+        linkUrl={
+          'https://join.slack.com/t/c4chi/shared_invite/zt-1mvwfme5d-0FtxC_4ZcGW9StKgfBvToQ'
+        }
+        imgUrl={Volunteers}
+        imageDescription={'Organization receiving help from Code for Chicago'}
+        imgLeft
+      />
+      <Heading
```
Comment (diorshelton) [inline src/pages/index.jsx:78]: Do we need a different button here linking the Meetup Group? This one is bigger than the component sizes.

File: src/pages/index.jsx [hunk 2]
```diff
@@ -1,21 +1,85 @@
...
-      <Card />
-      <Button text={'Submit'} small arrow />
-      <Button text={'Submit'} arrow />
-      <Button text={'Button'} secondary small />
-      <Button text={'Button'} secondary />
-      <Button text={'Link'} textBtn arrow link small />
-      <Button text={'Join Us on Slack'} textBtn arrow link />
+      <WorkWithUs>
+        <Heading
+          headingText="Work With Us"
```
Comment (diorshelton) [inline src/pages/index.jsx:41]: This text looks bolder in the Figma file 
https://www.figma.com/file/6ipIKBMMJL2TgWBzztA6lg/Handshake---Brigade-Design-System?node-id=1018%3A1059&t=mHbNkDyhYR5WiYdx-1

File: src/pages/index.jsx [hunk 3]
```diff
@@ -1,21 +1,85 @@
...
+          description="Whoever you are, there is a place for you here! Join us for co-working, project, service, and social events."
+        />
+        <IconGrid icons={icons} />
+      </WorkWithUs>
+
+      <ContentSection
+        background={'var(--blizzard-blue)'}
+        eyebrowText={'Organizations'}
+        mainHeading={
```
Comment (diorshelton) [inline src/pages/index.jsx:50]: Same here. The mainHeading text appears bolder in FIgma.

File: src/components/Footer.jsx [hunk 4]
```diff
@@ -98,29 +88,23 @@ const Footer = () => {
...
-              target="blank"
-              rel="noopener noreferrer"
-              className="button donateButton label-3"
-            >
-              Donate
-            </a>
-
-            <img src={longArrowAltRight} alt="" />
-          </div>
+          <Button
```
Comment (diorshelton) [inline src/components/Footer.jsx:91]: I may need to debug this button.  The wrong arrow appears here and on hover it blends into the background.

File: src/components/Footer.jsx [hunk 5]
```diff
@@ -98,29 +88,23 @@ const Footer = () => {
...
+            text="Donate"
+            link="https://www.codeforamerica.org/donate-to-a-brigade?utm_campaign=Code%20for%20Chicago&utm_source=Brigade%20site"
+            center
+            arrow
+          />
           <ul className="internal-links p2-body">
```
Comment (diorshelton) [inline src/components/Footer.jsx:97]: These links need the text decoration removed.

File: src/pages/index.jsx [hunk 6]
```diff
@@ -1,21 +1,85 @@
 import React from 'react'
-import Button from '../components/Button'
-import Card from '../components/Card'
-import Hero from '../components/Hero'
 import Layout from '../components/Layout'
+import Hero from '../components/Hero'
```
Comment (diorshelton) [inline src/pages/index.jsx:3]: There is extra spacing between the Nav and Hero

**Comment by diorshelton** [comment]
This is looking good Doug! I'm excited to see it all come together. I pointed out the things I saw when I compared it with  Figma.

File: src/components/Nav.jsx
Comment (daleinen7) [reply src/components/Nav.jsx:1]: Good call out.

File: src/pages/index.jsx (hunk 2 above)
Comment (daleinen7) [reply src/pages/index.jsx:41]: I made a separate ticket for this. Thanks for pointing it out.

File: src/components/Footer.jsx (hunk 5 above)
Comment (daleinen7) [reply src/components/Footer.jsx:103]: good catch

File: src/pages/index.jsx (hunk 6 above)
Comment (daleinen7) [reply src/pages/index.jsx:3]: another great catch, thanks!

**Pull request merged**